axelrod/tests/unit/test_tournament.py

```diff
@@ -279,25 +295,25 @@ def test_run_serial(self):
         calls = tournament._write_interactions.call_args_list
         self.assertEqual(len(calls), 15)
 
-    @unittest.skipIf(axelrod.on_windows,
-                     "Parallel processing not supported on Windows")
     def test_run_parallel(self):
+        class PickleableMock(MagicMock):
```

Comment:
I don't understand what this is doing. You haven't changed the actual parallel processing so this will make the tests pass but not fix the actual problem?